### PR TITLE
Ssbugs, removal of re-vive, and dashboard loop tick

### LINKED ISSUE #254: Supersmapling override bugs
Does not sync w/ toggle in regular settings.

Does not apply shown value when "enable manual supersampling override" on

Does not get the "default" value when it is off.

### LINKED ISSUE #273: Remove Revive menu and functionality
Related issue:
https://github.com/OpenVR-Advanced-Settings/OpenVR-AdvancedSettings/issues/86

Revive has directly deprecated support for OVRAS in the [release 1.8.0](https://github.com/LibreVR/Revive/releases/tag/1.8.0): `Revive: Removed support for the Revive tab in the old versions of OpenVR-AdvancedSettings`.

With this we can finally remove the Revive tab.

### PR DESCRIPTION
**Pull request by ykeara**
### Changes:

fixes #254 
fixes #273 

Adjusts how SS is reported and synced w/ steamVR to allow better clarity.... Also added a note denoting that you may need to refresh the overlay for it to take effect.

Added dashboardLoopTick() for fixfloor, settings, and steamVR to optimize what doesn't always need to be ran.

Removed Re-vive Tab Controller.

### Considerations:

The behavior is not quite ideal on SS (you can adjust slider before re-loading window), but I intend to move some of the settings around to a more logical.... I.E. supersampling will go to "Video" Tab among with some other changes. So its not overly important to fix at this point in time.

### CODE REVIEW

File: installer/installer.nsi [hunk 1]
```diff
@@ -167,5 +167,4 @@ Section "Uninstall"
 
 
 
-SectionEnd
-
+SectionEnd
```
Comment (username223) [inline installer/installer.nsi]: Unnecessary change?

File: src/tabcontrollers/SteamVRTabController.cpp [hunk 2]
```diff
@@ -9,7 +9,8 @@ void SteamVRTabController::initStage1()
 {
     initMotionSmoothing();
     initSupersampleOverride();
-    eventLoopTick();
+    // eventLoopTick();
```
Comment (username223) [inline src/tabcontrollers/SteamVRTabController.cpp]: Name of old function commented out.

File: src/tabcontrollers/SteamVRTabController.cpp [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #include "SteamVRTabController.h"
-#include <QQuickWindow>
 #include "../overlaycontroller.h"
+#include <QQuickWindow>
```
Comment (username223) [inline src/tabcontrollers/SteamVRTabController.cpp]: Switching around includes for no reason?

File: src/tabcontrollers/SteamVRTabController.cpp (hunk 2 above)
Comment (ykeara) [reply src/tabcontrollers/SteamVRTabController.cpp]: yep missed deleting that.

File: src/tabcontrollers/SteamVRTabController.cpp (hunk 3 above)
Comment (ykeara) [reply src/tabcontrollers/SteamVRTabController.cpp]: Will look into this TBH I didn't think i changed anything so I will need to look into this

**Comment by ykeara** [comment]
should have fixed those, the installer for .nsi might still say its different from previous but shouldn't be any issue..... I thin the .nsi was from some issues on my end I have no idea when I changed the order of the include statements tbh.

File: installer/installer.nsi [hunk 4]
```diff
@@ -168,4 +168,3 @@ Section "Uninstall"
 
 
 SectionEnd
```
Comment (username223) [inline installer/installer.nsi]: If you add a newline after `SectionEnd` this diff will disappear and I'll approve. :)
